examples/fl-dp-sa/fl_dp_sa/task.py

```diff
@@ -13,10 +13,10 @@
 
 DEVICE = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
```

Comment:
this shouldn't de defined as a global variable. It's better for simulations if clients "auto-discover" the device at run time, like this: https://github.com/adap/flower/blob/486cac3c5e68f717ad023e87e01bbb87c115c37d/examples/quickstart-pytorch/pytorchexample/client_app.py#L18